src/seedu/addressbook/Main.java

```diff
@@ -49,12 +49,16 @@ public void run(String[] launchArgs) {
      *
      */
     private void start(String[] launchArgs) {
-        try {
+        try {if(!doesFileExist()){
```

Comment:
and when we throw an exception, we don't need "try"

Try used when you "try to catch an exception"